lms/djangoapps/certificates/queue.py

```diff
@@ -147,19 +141,13 @@ def regen_cert(self, student, course_id, course=None, forced_grade=None, templat
                 )
                 return None
 
-            certificate.status = status.unavailable
-            certificate.save()
+            # invalidate the certificate, setting status to un
```

Comment:
Oops. I started to remove this comment but didn't fully delete it. I figured the comment was redundant. I plan on removing it.